experimental/framework/azure-client/src/interfaces.ts

```diff
@@ -11,45 +11,45 @@ import {
 import { ITelemetryBaseLogger } from "@fluidframework/common-definitions";
 import { ITokenProvider } from "@fluidframework/routerlicious-driver";
 
-export interface FrsContainerConfig {
+export interface AzureContainerConfig {
     id: string;
     logger?: ITelemetryBaseLogger;
 }
 
-export interface FrsConnectionConfig {
+export interface AzureConnectionConfig {
     tenantId: "local" | string;
     orderer: string;
     storage: string;
     tokenProvider: ITokenProvider;
 }
 
 /**
- * FrsContainerServices is returned by the FrsClient alongside a FluidContainer.
- * It holds the functionality specifically tied to the Frs service, and how the data stored in
+ * AzureContainerServices is returned by the AzureClient alongside a FluidContainer.
+ * It holds the functionality specifically tied to the Azure Relay Service, and how the data stored in
```

Comment:
I'm assuming it's for brevity here so the whole comment doesn't need to be reformatted, but is the full correct term still "Azure Fluid Relay service" as per the docs?

Reply:
Yep, updated to be the full term + abbreviation